packages/remix-eslint-config/rules/typescript.js

```diff
@@ -1,54 +1,49 @@
-const OFF = 0;
-const WARN = 1;
-const ERROR = 2;
```

Comment:
I'm intrigued by these constants. Why bother using them? ESLint already allows `"off"`, etc. - and errors if you provide an invalid config value.

Reply:
This has always confused me in the larger ecosystem as well.  I've long used the string values, and always assumed this was a pattern from early days of `eslint` when maybe string values weren't supported 🤷